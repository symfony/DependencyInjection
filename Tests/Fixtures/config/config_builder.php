<?php

use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
use Symfony\Component\DependencyInjection\Tests\Fixtures\AcmeConfig;

if ('prod' !== $env) {
    return;
}

return function (AcmeConfig $config, ContainerConfigurator $c) {
    $c->import('nested_config_builder.php');

    $config->color('blue');
};
