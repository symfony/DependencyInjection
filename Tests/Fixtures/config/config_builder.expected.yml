parameters:
    acme.configs: [{ color: red }, { color: blue }]

services:
    service_container:
        class: Symfony\Component\DependencyInjection\ContainerInterface
        public: true
        synthetic: true
